### PR TITLE
feat: [GTM-765]: Creating new endpoints for NG license CRUD

### PR DESCRIPTION
**Pull request by zhuoyin00**
You can use the following comments to re-trigger PR Checks

- Compile: `trigger compile`
- runAeriformCheck: `trigger AeriformCheck`
- CodeFormat: `trigger codeformat`
- MessageMetadata: `trigger messagecheck`
- Recency: `trigger recency`
- BuildNumberMetadata: `trigger buildnum`
- runDockerizationCheck: `trigger dockerizationcheck`
- runAuthorCheck: `trigger authorcheck`
- Checkstyle: `trigger checkstyle`
- PMD: `trigger pmd`
- TI-bootstrap: `trigger ti0`
- TI-bootstrap1: `trigger ti1`
- TI-bootstrap2: `trigger ti2`
- TI-bootstrap3: `trigger ti3`
- TI-bootstrap4: `trigger ti4`
- FunctionalTest1: `trigger ft1`
- FunctionalTest2: `trigger ft2`
- CodeBaseHash: `trigger codebasehash`

---
This change is [Reviewable](https://reviewable.io/reviews/harness/harness-core/31407)

### CODE REVIEW

File: 400-rest/src/test/java/software/wings/resources/AdminAccountResourceTest.java [hunk 1]
```diff
@@ -56,12 +57,14 @@
...
   @ClassRule
-  public static ResourceTestRule RESOURCES = ResourceTestRule.builder()
-                                                 .instance(new AdminAccountResource(adminAccountService,
-                                                     adminUserService, accessControlAdminClient, delegateService))
-                                                 .build();
+  public static ResourceTestRule RESOURCES =
+      ResourceTestRule.builder()
+          .instance(new AdminAccountResource(
+              adminAccountService, adminUserService, accessControlAdminClient, delegateService, adminLicenseHttpClient))
+          .build();
```
Comment (pranil-harness) [inline 400-rest/src/test/java/software/wings/resources/AdminAccountResourceTest.java:67]: It would be good to have new tests for the newly added REST Apis

**Review by zhuoyin00 — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/harness/harness-core/31407)* status: 2 file(s) to review. 1. unresolved discussions. 0 of 1 approvals obtained. 

___
*[400-rest/src/test/java/software/wings/resources/AdminAccountResourceTest.java, line 67 at r1](https://reviewable.io/reviews/harness/harness-core/31407#-My8prBgwbZJTWWe_pYm-r1-67:-MyDEgFl0gklR4wsyNWj:b-896fix) ([raw file](https://github.com/harness/harness-core/blob/0ae34260e05cef401a5f40a7a796b52b112c44e0/400-rest/src/test/java/software/wings/resources/AdminAccountResourceTest.java#L67)):*
<details><summary><i>Previously, pranil-harness wrote…</i></summary><blockquote>

It would be good to have new tests for the newly added REST Apis
</blockquote></details>

Done.

**Review by pranil-harness — APPROVED** [review]

**Review by pranil-harness — COMMENTED** [review]
> Reviewed 1 of 2 files at r1, 1 of 1 files at r2, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/harness/harness-core/31407)* status: 1. unresolved discussions.

**Review by pranil-harness — APPROVED** [review]
> *[Reviewable](https://reviewable.io/reviews/harness/harness-core/31407)* status: :shipit: complete! ✓

**Pull request merged**